src/main/java/pl/zaradny/springApp/exceptions/BadURLFormException.java

```diff
@@ -0,0 +1,8 @@
+package pl.zaradny.springApp.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.BAD_REQUEST, reason = "URL form is not correct")
+public class BadURLFormException extends RuntimeException {
```

Comment:
BadURLFormException -> Form? :)
Jeżeli chcesz walidować i robić bad request na podstawie złego url-a do obrazka, to warto stworzyć Exception BadImageUrlException
Czyli konkretnie tłumaczący gdzie i jaki jest problem